resources/shaders/fog.f.glsl

```diff
@@ -1,8 +1,20 @@
 #version 120
 
+varying vec2 v_lightValue;
+
+uniform int u_time;
+
 vec4 fog(vec4 color, float fogCoord, float fogStart, float fogEnd) {
 	float fog = clamp((fogEnd - fogCoord) / (fogEnd - fogStart), 0.0, 1.0);
 
-	return mix(vec4(0.196078, 0.6, 0.8, 1.0), color, fog);
+	const float pi = 3.1415927;
+	const float frequency = 480000;
+	// 960000
```

Comment:
Why is this comment here?

Reply:
comment isn't in my local file, so I probably pushed this and edited that out later, it's already removed (though it's the time limit, I pasted it here so I wouldn't lose it)